### PR TITLE
Amg 337 search fix

### PR DESCRIPTION
**Pull request by Antolius**

### CODE REVIEW

File: src/components/SearchBox/SearchBox.tsx [hunk 1]
```diff
@@ -120,6 +120,7 @@ export class SearchBox extends React.PureComponent<SearchBoxProps, SearchBoxStat
     const results = this.state.results.map(res => ({
       item: this.props.getItemById(res.meta)!,
       score: res.score,
+      matchData: res.matchData,
```
Comment (dario-ib) [inline src/components/SearchBox/SearchBox.tsx]: For what is matchData used?

File: src/services/SearchWorker.worker.ts [hunk 2]
```diff
@@ -40,13 +41,41 @@ const index: Promise<lunr.Index> = new Promise(resolve => {
...
 
-const expandTerm = term => '*' + lunr.stemmer(new lunr.Token(term, {})) + '*';
+const acronyms = {
+  sms: 'sms',
+  omni: 'omni',
+  ott: 'ott',
+  tfa: 'tfa',
+  '2fa': 'tfa',
+  otp: 'tfa',
+  totp: 'tfa',
```
[the remark this answers is not in the document]
Comment (dario-ib) [reply src/services/SearchWorker.worker.ts]: btw. what about some other acronyms? hlr, iot..

File: src/components/SearchBox/SearchBox.tsx (hunk 1 above)
Comment (Antolius) [reply src/components/SearchBox/SearchBox.tsx]: Just easier debugging. In tells you which search terms were matched in which of indexed fields (currently that Operation title and description for us).

It was easier for me to debug from React side (so in this component) than from service worker (which actually gets this info from lunr.js library).

File: src/services/SearchWorker.worker.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Antolius) [reply src/services/SearchWorker.worker.ts]: totp == Time-based One-Time Pin (like the one used in tfa)

I've added a few more acronyms I came up with.
Let's say that it for now. 😄

**Pull request merged**